ddtrace/internal/datadog/profiling/crashtracker/CMakeLists.txt

```diff
@@ -0,0 +1,121 @@
+cmake_minimum_required(VERSION 3.19)
```

Comment:
Are we supposed to build/test this and other cmake extensions from the top level directory?

Reply:
I have some infrastructure here for testing these native components independently of the normal pytest fixture, but that was only out of desperation.

I'd like to keep that system in place for now, but over the next several days I'll be adding tests to the normal system.

Reply:
I don't know whether that answers your question--sorry.

Reply:
I would prefer it to be part of the normal pytest. Even if it's not, I would be happy if there's an instruction on how to run these tests, like one liner command.

Reply:
Oh, I realize I didn't add the docs PR!  I'll add some docs--sorry about that, thank you

Reply:
I added two brief documents to`ddtrace/internal/datadog/profiling/docs` which explain some aspects of the build system.  In particular, they discuss how to use `build_standalone.sh` in order to do some useful things.